tools/libpfm/patch_headers.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+import sys
+import fnmatch
+import os
+
```

Comment:
I think this script can be replaced by something like this:

```
#!/bin/bash

cat <<EOF >> ._tmp_patterns
^#include <sys/types\.h>
^#include <ctype\.h>
[...]
EOF

files=$(find $DIR -name \*.h -or -name \*.c)

for f in $files; do
  mv "$f" "$f.bak"
  echo "KTF HEADER...." > "$f"
  grep -v -E -f ._tmp_patterns "$f.bak" >> "$f"
  rm "$f.bak"
done

rm -f ._tmp_patterns
```

Reply:
I would go with python script for now and later worry about it.
It's not actively used in build right now but later if we had to move towards new version of pfmlib, we can re-consider it.

Reply:
Ok, that makes sense.